src/autosuggest/__tests__/autosuggest-input.test.tsx

```diff
@@ -0,0 +1,264 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { act, Simulate } from 'react-dom/test-utils';
+import { render as renderJsx } from '@testing-library/react';
+import dropdownStyles from '../../../lib/components/internal/components/dropdown/styles.selectors.js';
+import { getFocusables } from '../../../lib/components/internal/components/focus-lock/utils';
+import createWrapper, { ElementWrapper } from '../../../lib/components/test-utils/dom';
+import AutosuggestInput, { AutosuggestInputRef } from '../../../lib/components/autosuggest/autosuggest-input';
+import { KeyCode } from '@cloudscape-design/test-utils-core/utils';
+import '../../__a11y__/to-validate-a11y';
```

Comment:
Looks like this import does nothing in this file

Reply:
Thanks, I will remove it from here. We check it on the autosuggest component which should be enough.